include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -40,30 +40,30 @@
 #endif
 
 #ifdef CONFIG_TI_CCFG_PRESENT
-  #define CCFG_SIZE 88
-  #define ROM_SIZE (CONFIG_FLASH_SIZE*1K - CONFIG_FLASH_LOAD_OFFSET - \
-		    CCFG_SIZE)
-  #define CCFG_ADDR (ROM_ADDR + ROM_SIZE)
+#define CCFG_SIZE 88
```

Comment:
> We should not use indentation for pre-processor directives.

Why not? IMO this reduces readability when we have a bunch of nested clauses

Reply:
This issue came up in another review; apparently nested pre-processor directives are not indented for ~90% of the code in Zephyr, so I wanted to do a step forward into this matter, and enforce consistency in the ARM linker script. If we agree to indent all nested #if and #define directives, we should have a rule for that....